src/lib/route-broadcaster.js

```diff
@@ -33,12 +35,16 @@ class RouteBroadcaster {
 
   * start () {
     yield this.crawlLedgers()
-    yield this.reloadLocalRoutes()
-    yield this.broadcast()
-    setInterval(() => this.routingTables.removeExpiredRoutes(), CLEANUP_INTERVAL)
+    try {
+      yield this.reloadLocalRoutes()
+      yield this.broadcast()
+    } catch (e) {
+      // This will be retried later
```

Comment:
No, you'd have to do something like this: https://github.com/interledger/five-bells-connector/blob/9e277604758222b7ad30787598159e2b2855a4a0/src/controllers/notifications.js#L94

Unless all Network related exceptions derived from the same base class.